std-bits/table/src/main/java/org/enso/table/data/column/storage/numeric/LongStorage.java

```diff
@@ -198,4 +187,70 @@ public LongStorage widen(IntegerType widerType) {
     assert widerType.fits(getType());
     return new LongStorage(data, (int) getSize(), getIsNothingMap(), widerType);
   }
+
+  /** Allow access to the underlying data array for copying. */
+  public long[] getArray() {
+    return data;
+  }
+
+  @Override
+  public ColumnLongStorageIterator iterator() {
+    return new LongStorageIterator(data, isNothing, (int) getSize());
+  }
+
+  private static class LongStorageIterator implements ColumnLongStorageIterator {
```

Comment:
Also `implements java.util.Iterator<Long>`!?

Reply:
It does.